index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'deep-match' {
```

Comment:
@fgnass fyi. if you are not familiar with TypeScript. This is TS declaration file - harmless to JavaScript projects and very useful for TypeScript projects.